recipe/meta.yaml

```diff
@@ -1,34 +1,39 @@
-{% set version = "6.5.3" %}
+{% set version = "6.6" %}
 
 package:
   name: cx_freeze
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/c/cx_Freeze/cx_Freeze-{{ version }}.tar.gz
-  sha256: e0d03cabcdf9b9c21354807ed9f06fa9481a8fd5a0838968a830f01a70820ff1
+  sha256: c4af8ad3f7e7d71e291c1dec5d0fb26bbe92df834b098ed35434c901fbd6762f
 
 build:
   skip: true  # [osx and py==39]
   number: 0
   script:
-    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+    - {{ PYTHON }} -m pip install cx_Logging --no-deps --ignore-installed --no-cache-dir -vv -i https://pypi.org/simple  # [win]
```

Comment:
@jschueller I need the dependent cx_logging. As it is not in conda and tried "pip install cx_Logging". But fail.
How I can bypass "Ignoring indexes: https://pypi.org/simple" by pip. It has an environment variable to change or what?

Reply:
Maybe it's best to create a package for it

Reply:
maybe it's one of these:
set "PIP_NO_BUILD_ISOLATION=False" 
set "PIP_NO_DEPENDENCIES=True" 
set "PIP_IGNORE_INSTALLED=True" 
set "PIP_CACHE_DIR=D:\bld\cx_freeze_1618505399456\pip_cache" 
set "PIP_NO_INDEX=True"

but maybe its best to create a dedicated package ?

Reply:
> but maybe its best to create a dedicated package ?

Yes, it is. Can you create?

Reply:
I'm under the water here, sorry

Reply:
Is this slang?